src/cli/java/org/commcare/util/screen/MultiSelectEntityScreen.java

```diff
@@ -1,14 +1,17 @@
 package org.commcare.util.screen;
 
 import static org.commcare.session.SessionFrame.STATE_MULTIPLE_DATUM_VAL;
+import static org.javarosa.core.model.instance.ExternalDataInstanceSource.buildVirtual;
```

Comment:
with the shorter name I makes sense to always call this with the class `ExternalDataInstanceSource.buildVirtual`